cmd/zstream/Makefile.am

```diff
@@ -1,3 +1,5 @@
+zstream_CPPFLAGS = $(AM_CPPFLAGS) $(FORCEDEBUG_CPPFLAGS)
```

Comment:
In the local branch I made that you beat me to this with, I had a comment here noting why it was Very Important this not be removed without more work, to avoid someone Chesterton's Fencing this later.

Might be useful to have one here.

Reply:
Commit added that doesn't quite do exactly this, but hopefully gets the vibe.

(also rare Chesterton reference in the wild warms my heart)